spec/views/civil_servants/civil_servants/edit.html.slim_spec.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'civil_servants/civil_servants/edit.html.erb', type: :view do
+  before do
+    sign_in user
+    assign(:civil_servant, civil_servant)
+
+    render
+  end
+
+  let(:civil_servant) { create(:civil_servant, :full) }
```

Comment:
You do not need to `create` when you're not doing queries in the view. This would improve spec performance

Reply:
I think it didn't work with build. But I will recheck for you.

Reply:
`ActionView::Template::Error: undefined method 'zdp' for nil:NilClass`